release: Bumped to version 5.0.0 (#207)
fauna/faunadb-python: README.rst

    print(indexes)

Exceptions handling
---------------

> Since 5.0.0 release, we introduced children classes for base HttpError class,
> for example:

Reviewer: ```suggestion
With the 5.0.0 release, child classes of the base HttpError class now exist.
For example:

```

```
class PayloadTooLarge(HttpError):

>     pass

Reviewer: Including `pass` here (and in subsequent examples) seems inappropriate. Perhaps just list the exception types and a brief description for each?

class InvalidArgumentError(HttpError):
    pass

class InvalidExpressionError(HttpError):
    pass

class InvalidUrlParameterError(HttpError):
    pass

class SchemaNotFoundError(HttpError):
    pass
```

> where each class corresponds to the error code from fauna database server,

Reviewer: ```suggestion
Each class corresponds to an error code from Fauna.
```

> you can inspect "errors.py" file for more information on how it is implemented.

Reviewer: ```suggestion
Inspect the [`errors.py`](https://github.com/fauna/faunadb-python/blob/v5/faunadb/errors.py)
file for more information on how it is implemented.
```

> Below you can see an example of how you can take advantage of a new exception hierarchy:

Reviewer: ```suggestion
The following example demonstrates the methods that you can access from the exception object:
```
